plugins/cgroups/slurm/README.md

```diff
@@ -54,6 +54,35 @@ For instance, the `slurm` configuration section should be after the `process-to-
 …
 ```
 
+## Levels of surveillance
```

Comment:
I think that ["surveillance"](https://www.wordreference.com/enfr/surveillance) is not the right word for what we're doing.